welcome/links-documentation.md

```diff
@@ -0,0 +1,16 @@
+# Links & Documentation
```

Comment:
because everything else is german, you should also write Documentation in german